src/Nethermind/Nethermind.Synchronization/ParallelSync/MultiSyncModeSelector.cs

```diff
@@ -608,8 +607,7 @@ private bool ShouldBeInStateSyncMode(Snapshot best)
                     (nameof(hasFastSyncBeenActive), hasFastSyncBeenActive),
                     (nameof(hasAnyPostPivotPeer), hasAnyPostPivotPeer),
                     ($"{nameof(notInFastSync)}||{nameof(stickyStateNodes)}", notInFastSync || stickyStateNodes),
-                    (nameof(stateNotDownloadedYet), stateNotDownloadedYet),
-                    (nameof(longRangeCatchUp), longRangeCatchUp),
+                    ($"{nameof(stateNotDownloadedYet)}||{nameof(longRangeCatchUp)}", stateNotDownloadedYet || longRangeCatchUp),
```

Comment:
```suggestion
                    ($"{nameof(stateNotDownloadedYet)} || {nameof(longRangeCatchUp)}", stateNotDownloadedYet || longRangeCatchUp),
```

change the one above too?